src/test-utils/setup-tests.js

```diff
@@ -1,7 +1,4 @@
 import '@testing-library/jest-dom'
-import { configure } from 'enzyme'
```

Comment:
I got errors about TextEncoder being undefined after the update of cli-app-scripts (which seems to be a jest-dom v16 issue), but this setup was copied from another app and isn't needed, so I've just removed it to work around that issue.